javascript/react/src/controls/SortInstruction.tsx

```diff
@@ -14,6 +16,13 @@ import { Container, icons } from './SortInstruction.styles.js'
 import type { SortInstructionProps } from './SortInstruction.types.js'
 import { TableColumnDropdown } from './TableColumnDropdown.js'
 
+const narrowerStyles = merge({}, dropdownStyles, {
```

Comment:
Is the empty object even necessary here?

Reply:
yeah, lodash merge considers the left-most object the "target", so it gets mutated. This reminds me I meant to move this out to a reusable style though since the same thing is used in a couple of places